di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/config/CoreStubConfig.java

```diff
@@ -42,6 +42,9 @@ public class CoreStubConfig {
             getConfigValue("CORE_STUB_CONFIG_FILE", "/app/config/cris-dev.yaml");
     public static final String CORE_STUB_SIGNING_PRIVATE_KEY_JWK_BASE64 =
             getConfigValue("CORE_STUB_SIGNING_PRIVATE_KEY_JWK_BASE64", null);
+
+    public static final String CORE_STUB_SIGNING_PUBLIC_JWK =
```

Comment:
Does the README or any other files need to be updated to set the environment variable `CORE_STUB_SIGNING_PUBLIC_JWK` ?

Reply:
The value comes from di-ipv-config see `https://github.com/alphagov/di-ipv-config/blob/main/stubs/di-ipv-core-stub/.env`